src/Mod/Fem/femsolver/calculix/solver.py

```diff
@@ -372,6 +372,16 @@ def add_attributes(obj, ccx_prefs):
         obj.BeamShellResultOutput3D = dimout
 
 
+    if not hasattr(obj, "BeamReducedIntegration"):
+        obj.addProperty(
+            "App::PropertyBool",
+            "BeamReducedIntegration",
+            "Fem",
+            "Set to True to use beam elements with reduced integration"
+        )
+        red = ccx_prefs.GetBool("BeamReducedIntegration", True)
+        obj.BeamReducedIntegration = red
```

Comment:
You are trying to use the preference parameter `BeamReducedIntegration`, but it does not exist.
Usually you should associate a check box on the preferences page to that parameter.
If you don't want to relate the property to a preference parameter, just add the property without referencing a parameter and set the default value as `obj.BeamReducedIntegration = True`